src/encoded/audit/experiment.py

```diff
@@ -4200,9 +4200,12 @@ def audit_biosample_perturbed_mixed(value, system, excluded_types):
 def check_experiment_atac_encode4_qc_standards(experiment, files_structure):
     # https://encodedcc.atlassian.net/browse/ENCD-5255
     # https://encodedcc.atlassian.net/browse/ENCD-5350
+    # https://encodedcc.atlassian.net/browse/ENCD-5468
     alignment_files = files_structure.get('alignments').values()
     stable_peaks_files = files_structure.get('stable_peaks_files').values()
-    atac_peaks_files = []
+    idr_thresholded_peaks_files = files_structure.get('idr_thresholded_peaks').values()
+    audit_idr_thresholded_peaks_files = []
```

Comment:
I am not sure about the prefix "audit" being used for the variable name. Maybe "idr_thresholded_peak_files_to_audit"? I am thinking of someone reading the code and trying to decipher the meaning of these variable names

Reply:
Thanks, I renamed this as suggested